src/pages/meetups/index.js

```diff
@@ -16,6 +17,8 @@ const Meetups = ({ data }) => {
       meetup.childMdx.frontmatter.meetup_date !== null
   );
 
+  return <RedirectCodeursEnSeine path="/meetups" />;
```

Comment:
There is no other way to handle this at "Gatsby level"? Because here of there is a link from CES part to a sub page of meetup it will not be handled.

Reply:
I understand what you mean, but even the plugins at Gatsby level do not take any REGEX or GLOB in the URL to take into account and that way, we have to put as many redirect as there is for the subpages.